Source/ACE/Network/Enum/Spell.cs

```diff
@@ -8,10 +8,6347 @@ namespace ACE.Network.Enum
 {
     public enum Spell : uint
     {
-        FlameBolt           = 0x1B,
-        FrostBolt           = 0x1C,
-        LightningBolt       = 0x4B,
-        ForceBolt           = 0x56,
-        WhirlingBlade       = 0x5C,
+        Undef,
```

Comment:
The values are equal to what is in the SpellTable. The enum'd names come from the client, up to the PDB version we have. (The rest were approximated to what they likely would've been, so at least they were visible in aclogview). Either we can read the SpellTable directly if you want them, or I've got a CSV dump of it if you want to enum it somehow other than this.